client/src/components/organisms/style/AdminRequestGroupList.scss

```diff
@@ -0,0 +1,57 @@
+.title {
+    font-family: Inter;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    letter-spacing: 0.2px;
+
+    color: #333333;
+
+    padding-left: 95px;
+}
+
+.row {
+    display: flex;
+    justify-content: space-between;
+}
+
+.action-group {
+    display: flex;
+    flex-direction: row;
+}
+
+.action-group .spacing {
+    width: 27px;
+}
+
+.button {
+    @include normal-button;
+    width: -webkit-fit-content;
+    width: -moz-fit-content;
+    width: 160px;
+    height: 40px;
+    display: flex-end;
+
+    background: #004D57;
+    border-radius: 4px;
+
+    font-family: Inter;
```

Comment:
Include mixin from common font styling

Reply:
> > @guy-ro do you have any idea how to disable page navigation when there are no results?
> 
> Regarding this, conditionally (on props.requestGroups.length == 0), set visibility of the pagination to hidden (CSS)

@guy-ro looking at the figma, i think when there are no request groups, pagination shouldn't be entirely hidden, it should just show two disabled arrows (< >) without any page number info. if you go to "Groups Table / Empty" on the figma, you'll see what i mean. so i think this should be a change within the SimplePageNavigation component. do you have any ideas how to do it? if not, i can also take a look